benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -52,10 +52,32 @@ export default class SelectColumn extends React.Component {
   };
 
   renderColumns = index => {
-    return this.state.list[index].columns.map(column => (
+    const idxRow = [];
```

Comment:
I still don't understand why the array `idxRow` is used. It is only filled and then immediately used for iterating over it. Why not get rid of it?

Reply:
`idxRow` is removed. I used `self.handleSelecion` instead of `this.handleSelecion` because it leads to an error. Now empty cells are empty `<td>` with keys.